configs/example/apu_se.py

```diff
@@ -892,9 +892,9 @@ def find_file(base_list, rel_path):
 token_port_idx = 0
 for i in range(len(system.ruby._cpu_ports)):
     if isinstance(system.ruby._cpu_ports[i], VIPERCoalescer):
-        system.cpu[shader_idx].CUs[token_port_idx].gmTokenPort = (
-            system.ruby._cpu_ports[i].gmTokenPort
-        )
+        system.cpu[shader_idx].CUs[
```

Comment:
This will need to be removed

Reply:
We do not want this change? I think the reason it warns me about this is its black format issue.

Reply:
I don't think this change has anything to do with the rest of this commit, right?

Reply:
Assuming that's the case, then if this change is needed, we should make a separate pull request for it

Reply:
It is irrelevant to our change, and I can make a separate pull request for it

Reply:
Yes, please do -- thanks!

Reply:
I just found out that the latest develop already solved that issue, and I have already merged develop into this branch